job-dsls/jobs/prod/prod_cron_meta_nightly_pipeline.groovy

```diff
@@ -30,6 +30,9 @@ def KOGITO_BLUE_NEXT_PRODUCT_VERSION='1.13.2.blue'
 def KOGITO_BLUE_NEXT_PRODUCT_BRANCH='1.13.x-blue'
 def KOGITO_BLUE_NEXT_PRODUCT_CONFIG_BRANCH="kogito/1.13.x-blue"
 
+def SERVERLESS_LOGIC_NEXT_PRODUCT_VERSION='999.0.0'
```

Comment:
This is now required because the upstream version is 999 (without .0.0 in the end) and before it used to be 2.0.0-SNAPSHOT, so we need this otherwise PME is not able to properly deal with the versions in the nightly.